### PR TITLE
set manip imporvements

### PR DESCRIPTION
**Pull request by Okm165**

### CODE REVIEW

File: scarb/src/bin/scarb/args.rs [hunk 1]
```diff
@@ -226,8 +226,12 @@ pub struct BuildArgs {
     pub test: bool,
 
     /// Which features to enable in code.
-    #[arg(short, long)]
-    pub features: Option<String>,
+    #[arg(short, long, value_delimiter = ',')]
```
Comment (fmkra) [inline scarb/src/bin/scarb/args.rs:229]: Is it possible to add multiple delimiters? For instance, we want to support separating features with either comma or space.

**Review by fmkra — APPROVED** [review]

File: scarb/src/bin/scarb/args.rs (hunk 1 above)
Comment (Okm165) [reply scarb/src/bin/scarb/args.rs:229]: i think it is not possible
function declaration expects char here
https://docs.rs/clap_builder/4.5.2/src/clap_builder/builder/arg.rs.html#1546-1551

Comment (Okm165) [reply scarb/src/bin/scarb/args.rs:229]: so we need to decide if `,` or ` ` to use as delimeter

Comment (fmkra) [reply scarb/src/bin/scarb/args.rs:229]: Ok so let's leave it as is

**Pull request merged**